addons/l10n_ph/data/template/account.tax-ph.csv

```diff
@@ -1,121 +1,153 @@
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201","+42SB"
+"","","","","","","","","","","","","100","base","refund","","-42SA"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201","-42SB"
+"l10n_ph_tax_purchase_vat_exempt","20","0% EXEMPT","","True","VAT Exempt","purchase","0.0","percent","l10n_ph_tax_group_vat_exempt","","","100","base","invoice","","+46E"
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201",""
+"","","","","","","","","","","","","100","base","refund","","-46E"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201",""
+"l10n_ph_tax_purchase_vat_exempt_import","20","0% EXEMPT I","","False","VAT Exempt Imports","purchase","0.0","percent","l10n_ph_tax_group_vat_exempt","","","100","base","invoice","","+47A"
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201",""
+"","","","","","","","","","","","","100","base","refund","","-47A"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201",""
+"l10n_ph_tax_purchase_vat_zero_rated","20","0% ZR","","True","Zero Rated","purchase","0.0","percent","l10n_ph_tax_group_vat_exempt","","","100","base","invoice","","+46ZR"
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201",""
+"","","","","","","","","","","","","100","base","refund","","-46ZR"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201",""
+"l10n_ph_tax_purchase_vat_12_service_non_residents","20","12% S NR","","False","12% VAT Non Resident","purchase","12.0","percent","l10n_ph_tax_group_vat_12","service","","100","base","invoice","","+43A"
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201","+43B"
+"","","","","","","","","","","","","100","base","refund","","-43A"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201","-43B"
+"l10n_ph_tax_purchase_vat_12_capital","20","12% C","","True","12% Capital","purchase","12.0","percent","l10n_ph_tax_group_capital_12","consu","","100","base","invoice","","+CAPA"
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201","+CAPB"
+"","","","","","","","","","","","","100","base","refund","","-CAPA"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201","-CAPB"
+"l10n_ph_tax_purchase_vat_12_import","20","12% I","","False","12% Imports","purchase","12.0","percent","l10n_ph_tax_group_capital_12","consu","","100","base","invoice","","+44A"
+"","","","","","","","","","","","","100","tax","invoice","l10n_ph_110201","+44B"
+"","","","","","","","","","","","","100","base","refund","","-44A"
+"","","","","","","","","","","","","100","tax","refund","l10n_ph_110201","-44B"
+"l10n_ph_tax_purchase_wi010","30","5% P F","5% WI010 - Prof Fees","True","Prof Fees","purchase","-5.0","percent","l10n_ph_tax_group_wht_5","","WI010","100","base","invoice","",""

```

Comment:
Don't know if it's functionally what we want or not but we don't have `repartion_line_ids/tag_ids` value since this line

Reply:
Yes, withholding taxes are not reported in a way we support in Odoo atm, so they have been left out for now.